Prox/Prox/PlaceDetails/FilterViewController.swift

```diff
@@ -103,20 +112,21 @@ class FilterViewController: UIViewController {
             container.leadingAnchor.constraint(equalTo: view.leadingAnchor),
 
             placeCountLabel.centerXAnchor.constraint(equalTo: container.centerXAnchor),
+            placeCountLabel.centerYAnchor.constraint(equalTo: closeButton.centerYAnchor),
 
-            closeButton.centerYAnchor.constraint(equalTo: placeCountLabel.centerYAnchor),
-            closeButton.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant: -spacing),
+            closeButton.topAnchor.constraint(equalTo: view.topAnchor, constant: 45),
```

Comment:
Should this be `container.topAnchor`? I'm concerned this will continue to draw (underneath the photo carousel?) if it's constrained to the outer outer view.